concordium-sdk/src/main/java/com/concordium/sdk/responses/peerList/PeerCatchupStatus.java

```diff
@@ -0,0 +1,58 @@
+package com.concordium.sdk.responses.peerList;
+
+import concordium.ConcordiumP2PRpc;
+
+/**
+ * Represents Catchup Status of a Node.
+ */
+public enum PeerCatchupStatus {
+
+    /**
+     * Node has the latest updates.
+     */
+    UPTODATE(0),
+
+    /**
+     * Node updates are pending.
+     */
+    PENDING(1),
+
+    /**
+     * Node is downloading new updates.
+     */
+    CATCHINGUP(2),
+
+    /**
+     * Unknown status.
+     */
+    UNRECOGNIZED(-1);
```

Comment:
Please remove this and throw an `IllegalArgumentException` with a sufficient error message in `parse` below.

Reply:
done